doc/changelogs/CHANGELOG_V17.md

```diff
@@ -44,6 +45,108 @@
+* \[[`cbdc0ab9c7`](https://github.com/nodejs/node/commit/cbdc0ab9c7)] - **bootstrap**: run inspector and event loop in snapshot builder (Joyee Cheung) [#42466](https://github.com/nodejs/node/pull/42466)
+* \[[`fa28439583`](https://github.com/nodejs/node/commit/fa28439583)] - **bootstrap**: make I/O streams work with user-land snapshot (Joyee Cheung) [#42466](https://github.com/nodejs/node/pull/42466)
+* \[[`a113d4379a`](https://github.com/nodejs/node/commit/a113d4379a)] - **bootstrap**: use SnapshotData to pass snapshot data around (Joyee Cheung) [#42360](https://github.com/nodejs/node/pull/42360)
+* \[[`3fbfd6550d`](https://github.com/nodejs/node/commit/3fbfd6550d)] - **buffer**: improve Blob constructor error message when passing a string (Xuguang Mei) [#42338](https://github.com/nodejs/node/pull/42338)
+* \[[`b250b8ec95`](https://github.com/nodejs/node/commit/b250b8ec95)] - **buffer**: fix `atob` input validation (Antoine du Hamel) [#42539](https://github.com/nodejs/node/pull/42539)
+* \[[`ebefd25141`](https://github.com/nodejs/node/commit/ebefd25141)] - **build**: consolidate JS and md linting GitHub Actions (Rich Trott) [#42572](https://github.com/nodejs/node/pull/42572)
+* \[[`fe62d4b506`](https://github.com/nodejs/node/commit/fe62d4b506)] - **build**: set stale action back to running nightly (Michael Dawson) [#42549](https://github.com/nodejs/node/pull/42549)
+* \[[`6b14bd8f31`](https://github.com/nodejs/node/commit/6b14bd8f31)] - **build**: add --node-snapshot-main configure option (Joyee Cheung) [#42466](https://github.com/nodejs/node/pull/42466)
+* \[[`b647336c96`](https://github.com/nodejs/node/commit/b647336c96)] - **build**: bump actions/checkout (Eliaz Bobadilla) [#42460](https://github.com/nodejs/node/pull/42460)
+* \[[`7105f8b467`](https://github.com/nodejs/node/commit/7105f8b467)] - **child\_process**: add env contents types in JSDoc (Rich Trott) [#42494](https://github.com/nodejs/node/pull/42494)
+* \[[`fa7edb7acd`](https://github.com/nodejs/node/commit/fa7edb7acd)] - **crypto**: do not add undefined hash in webcrypto normalizeAlgorithm (Filip Skokan) [#42559](https://github.com/nodejs/node/pull/42559)
+* \[[`c036258809`](https://github.com/nodejs/node/commit/c036258809)] - **crypto**: cleanup webcrypto jwk code (Filip Skokan) [#42562](https://github.com/nodejs/node/pull/42562)
+* \[[`4e7b7465bc`](https://github.com/nodejs/node/commit/4e7b7465bc)] - **crypto**: fix webcrypto derive key lengths (Filip Skokan) [#42542](https://github.com/nodejs/node/pull/42542)
+* \[[`2ca08c948c`](https://github.com/nodejs/node/commit/2ca08c948c)] - **(SEMVER-MINOR)** **crypto**: make authTagLength optional for CC20P1305 (Tobias Nießen) [#42427](https://github.com/nodejs/node/pull/42427)
+* \[[`9158b91440`](https://github.com/nodejs/node/commit/9158b91440)] - **deps**: update undici to 4.16.0 (Node.js GitHub Bot) [#42414](https://github.com/nodejs/node/pull/42414)
```

Comment:
I think these two could be notable

Reply:
Sure! I will put them!

I got the "notable changes" from the guidelines, from "branch-diff"

Reply:
Updated!